### PR TITLE
[wip] Pytest 2.8.4

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
started to prepare the release, will follow the docs and use the pr to track the progress + gather notes

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
i'm skipping the plugin index update since the script breaks in a incomprehensible manner for me and i will defer investigation until after the release

File: doc/en/announce/release-2.8.4.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+pytest-2.8.3
```
Comment (nicoddemus) [inline doc/en/announce/release-2.8.4.rst]: Should be `2.8.4`

File: doc/en/announce/release-2.8.4.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+============
+
+pytest is a mature Python testing tool with more than a 1100 tests
+against itself, passing on many different interpreters and platforms.
+This release is supposed to be drop-in compatible to 2.8.2.
```
Comment (nicoddemus) [inline doc/en/announce/release-2.8.4.rst]: Should be `2.8.3`

**Comment by nicoddemus** [comment]
Other than my comments above, :+1: to ship it! :smile:

**Pull request merged**